types/web3/index.d.ts

```diff
@@ -50,5 +51,3 @@ declare class Web3 {
     setProvider(provider: Provider): void;
     utils: Utils;
 }
-
-export = Web3;
```

Comment:
I fear this is important. Web3 uses `module.exports` https://github.com/ethereum/web3.js/blob/326e42fe3687d3e34840dbe4797408dfe6364917/packages/web3/src/index.js#L78